### PR TITLE
adding travis to the staging branch

### PR DESCRIPTION
**Pull request by developer0623**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  yarn: true
+  directories:
+    - node_modules
+
+install:
+  - yarn
+
+script:
+  - node --version
+  #- yarn lint
```
Comment (vishakh) [inline .travis.yml]: Why is the linter turned off?

File: package.json [hunk 2]
```diff
@@ -37,7 +37,8 @@
     "libsodium-wrappers": "^0.5.4",
     "libsodium-wrappers-sumo": "^0.5.4",
     "tsconfig-paths-webpack-plugin": "^3.2.0",
-    "zxcvbn": "^4.4.2"
+    "zxcvbn": "^4.4.2",
+    "yarn": "^1.10.1"
```
Comment (vishakh) [inline package.json]: How come we are using Yarn now?

Comment (anonymoussprocket) [reply package.json]: this matches master
we can rip it out in the next PR

File: .travis.yml (hunk 1 above)
Comment (developer0623) [reply .travis.yml]: There is no `lint` script, yet.
But I think it will be necessary after.

File: package.json (hunk 2 above)
Comment (developer0623) [reply package.json]: @vishakh We can use both of npm and yarn.

**Comment by umurb** [comment]
Switching to alphanet as the active development environment. All ConseilJS development should be made against develop branch from now on. Since develop already has Travis configured, I am closing this PR

**Pull request closed without merging**